pandas/io/tests/test_parsers.py

```diff
@@ -4652,6 +4655,9 @@ def test_invalid_c_parser_opts_with_not_c_parser(self):
         engines = 'python', 'python-fwf'
         for default in c_defaults:
             for engine in engines:
+                if 'python' in engine and default not in py_unsupported:
```

Comment:
ok another PR is fine.

ideally we have some tests which will explicity test for the unsupported features (so that when one IS defined in say the python-parser that was previously only in the c-parser) it will force an explict change (in the PR).